### ISSUE TITLE
Priority of mcontrol/mcontrol6 load data before trigger

### ISSUE THREAD

**Issue by en-sc** [riscv/riscv-debug-spec#985]
There are two entries in table 10 (5.3. Priority)

![image](https://github.com/riscv/riscv-debug-spec/assets/109669442/dc8191e6-3988-4ce2-9d6a-1d34bbdc58f7)

and

![image](https://github.com/riscv/riscv-debug-spec/assets/109669442/a69a8af7-81ec-4c6e-b9fe-b8f0255df985)

AFAIK, both these entries specify the priority for `mcontrol/mcontrol6 load data before` (the second one explitly, the first one as one of the options).
Is this a typo? If not, what is the priority for `mcontrol/mcontrol6 load data before`?

**Comment by pdonahue-ventana** [comment]
Yes, this is not accurately specified.  It has the lowest priority and shouldn't be included with the rest of them in the middle of that table.

Load/store address and store data are known before doing any memory access.  Therefore, they can have a relatively high priority (which is the first thing you show).  Load data cannot possibly be known until the load has been performed.  That must happen after address translation and PMP checks.  So load data before must have the lowest priority.

Nobody probably caught this because load data should (but not must) fire after instead of before.  See table 11.
